dash_docs/chapter_index.py

```diff
@@ -362,6 +362,13 @@ def component_list(
                     'description': 'Use a staged Dash App to test changes before updating '
                     'your production Dash App.'
                 },
+                {
+                    'url': '/dash-enterprise/database-connections',
+                    'content': chapters.dash_enterprise.dash_enterprise_chapters.DataConnections,
+                    'name': 'Dash App SQL Database Connections',
```

Comment:
Given that most of the other entries in this section appear to include action verbs to start, what if we did something similar here?

```suggestion
                    'name': 'Connect a Dash App to an SQL Database',
```